lib/tasks/impersonation_admin_user.rake

```diff
@@ -2,17 +2,21 @@
 
 namespace :impersonation_admin_user do
   task :update, %i[username domain] => :environment do |_task, args|
-    update_query = if System::Database.oracle?
-                     "username = :username, email = :username || '+' || accounts.self_domain || '@' || :domain"
-                   else
-                     "username = :username, email = CONCAT(:username, '+', accounts.self_domain, '@', :domain)"
-                   end
+    username = args[:username]
+    domain = args[:domain]
 
-    updated_records = User
-      .where(username: '3scaleadmin')
-      .joins(:account).references(:account)
-      .update_all([update_query, username: args[:username], domain: args[:domain]])
-
-    puts "Updated #{updated_records} impersonation_admin users"
+    db_specific = if System::Database.oracle?
+                    "'#{username}' || '+' || accounts.self_domain || '@' || '#{domain}'"
+                  else
+                    "CONCAT('#{username}', '+', accounts.self_domain, '@', '#{domain}')"
```

Comment:
This is not escaped, so vulnerable to SQLi and generally discouraged. 

At least you have to use https://api.rubyonrails.org/v4.2.10/classes/ActiveRecord/ConnectionAdapters/Quoting.html#method-i-quote_string to properly escape it.